automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath 
```

Comment:
We would need to add a check here to see if the bindir actually exist, the user may have overridden the install directory.
Best option would be to use the [Get-AppInstallLocation](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/extensions/chocolatey-core.extension/extensions/Get-AppInstallLocation.ps1) available from the `chocolatey-core.extenion` package (which we already have a dependency on), and get the installed directory that way (of course, it needs to be tested that it actually exist).